src/duplocloud/client.py

```diff
@@ -316,7 +316,7 @@ def logger_for(self, name: str=None) -> logging.Logger:
     logger = logging.getLogger(name)
     lvl = logging.getLevelName(self.loglevel)
     logger.setLevel(lvl)
-    formatter = logging.Formatter("%(levelname)s %(message)s")
+    formatter = logging.Formatter("%(message)s")
```

Comment:
Pretty sure this will affect all of the logs, not just the pod logs. I would prefer having a seperate logger for the pod logs. Then a logger for normal logs.